_pages/about-us/tts-history.md

```diff
@@ -12,6 +12,19 @@ While OPP was paving the way for government shared services and open data, there
 
 That project became [18F](https://18f.gsa.gov/) when the office officially launched in March 2014. 18F quickly grew in size to help federal agencies work on technology projects while learning new skills. In 2016, 18F, PIF, and OPP came together under the newly-formed Technology Transformation Service office. In 2017, TTS added the [IT Modernization Centers of Excellence (CoEs)](https://coe.gsa.gov/) to focus on whole-agency technology modernization, and all of TTS moved into GSA’s existing Federal Acquisition Service.
 
+
+## TTS Today
+
+TTS is part of GSA's Federal Acquisition Service (FAS) and contains several offices:  
+
+- **18F** partners with agencies to provide consulting and software development services, with a focus on cultural transformation and empowerment. A critical part of 18F’s mission is to teach agencies new digital skills during projects so they can take over continued development.
+- **The Office of Products and Programs (OPP)** houses a variety of products and services such as USA.gov, Data.gov, Challenge.gov and the Digital Analytics Program that either directly serve the public or empower other government agencies.
+- **The Centers of Excellence (CoEs)** partner with agencies on specific verticals in order to drive IT modernization in the government by leveraging private sector innovation, existing government services, and providing access to best practices and expertise.
+- **The Presidential Innovation Fellowship (PIF)**  is a year-long fellowship made up of mid- to senior-level individuals with a proven track record of success in entrepreneurship, strategic design, technology, innovation, and modern product management, who pair with agencies to tackle the most pressing technology challenges.
+- **TTS Operations** keep everything running and ensures the team has the tools needed to focus on the TTS mission. This includeskeeping our finances straight, making sure we’re in compliance with all the rules we have to follow, helping us go to conferences, and recruiting new employees, among other things.
+- **Office of Acquisition** offers acquisition services and buys digital products and services for our own organization, TTS, while  demonstrating that compliance can be more efficient and effective. In the spirit of innovation, OA experiments with new, collaborative ways of doing procurement while adhering to the all regulations and policies.
+
```

Comment:
**This is what we currently share with new hires:**

`18F (Services company and product incubator)` 
Provides digital development and consulting services for any government agency or program
`Office of Acquisitions (AcqStack)` 
Drives smarter technology procurement through technical procurement consultations, new procurement vehicle creation, and the development of experimental procurement platforms and methodologies.  Oversee TTS’ buying component of acquisitions
`Presidential Innovation Fellowship (PIF)` 
Pairs talented, diverse technologists and innovators with top civil-servants and change-makers in the federal government to tackle some our nation’s biggest challenges
`Product company (OPP)` 
The Office of Products and Programs helps agencies improve delivery of information and services to the public.  Responsible for operations, evolution, and scaling of mature products and services that fall under five mission areas
`Centers of Excellence` 
Accelerate IT modernization to improve the citizen experience, improve outcomes and reduce legacy IT spending across the government.
`Office of Operations`
Internal support team comprised of Talent, Business Operations, and Outreach

**Not sure if this is pertinent to what you're working on, but I just wanted to get some alignment on it so new hires are receiving consistent messages.**

Reply:
Hannah, I've updated the text based on this. Will you take another look?